dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -1413,4 +1429,81 @@ public final Response bulkUnarchive(@Context final HttpServletRequest  request,
                         new BulkResultView(unarchivedContainersCount,0L,failedToUnarchive)))
                 .build();
     }
-}
+
+    /**
+     * Saves a new working version of a container.
+     *
+     * @param request
+     * @param response
+     * @param containerForm
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @POST
+    @Path("_add")
```

Comment:
nor sure if it is need
POST containers should be ok

Reply:
Since we are not going to define this endpoint in v2 and in the v1 containerResource we already have 

 @POST
    @JSONP
    @NoCache
    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
    public final Response saveNew(@Context final HttpServletRequest  request,
                                  @Context final HttpServletResponse response,
                                  final ContainerForm containerForm)



therefore there maybe ambiguity to identify correct endpoint - WDYT ?

Reply:
this endpoint has been removed and existing endpoint to create new container has been modified with little code changes to save code field whrn maxContentlet is 0 @jdotcms @freddyucv